test/unittests/parser_expr_test.cpp

```diff
@@ -15,7 +15,7 @@ namespace
 {
 inline auto parse_expr(const bytes& input)
 {
-    return fizzy::parse_expr(input.data(), input.data() + input.size(), false);
+    return fizzy::parse_expr(input.data(), input.data() + input.size(), Module{});
```

Comment:
Is there any other use case for `Module` here? I would rather not pass it around for just the `has_memory` flag.

Reply:
It is later used to access functions' types to stack height around calls.

Reply:
Yes, realised that after checking other commits.